src/components/uielements/assets/assetInput/assetInput.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { tokenAmount } from '@thorchain/asgardex-token'
```

Comment:
`@thorchain/asgardex-token` has been deprecated. Use `@thorchain/asgardex-util`, which includes all of `asgardex-token`, but renamed its function from `tokenXXX` to `assetXXX`